docs/getting-started/system-requirements.md

```diff
@@ -31,3 +31,103 @@ Beginning with Highcharts v11, legacy browsers must load Highcharts from the `es
 ```
 
 One of the legacy browsers requiring the files from the `es5` folder is QtWeb (wkhtmltopdf's QT Webkit rendering engine is based on this browser).
+
+Other possible adjustments need:
+
+- Missing ployfill:
+
+    If the chart doesn't show and console errors pop-up mentioning that "include" is missing, it's more than likely that polyfills aren't loaded. These can be added manually **before** any highcharts module is loaded. The example below shows necessary polyfills and when to load them.
+
+    Polyfills:
+
+    ```html
+    <!-- Load polyfills before any highcharts scripts are loaded. -->
+    <script>
+        if (!Array.prototype.includes) {
+            // eslint-disable-next-line no-extend-native
+            Array.prototype.includes = function (searchElement, fromIndex) {
+                return this.indexOf(searchElement, fromIndex) > -1;
+            };
+        }
+        if (!Array.prototype.find) {
+            // eslint-disable-next-line no-extend-native
+            Array.prototype.find = function (predicate, thisArg) {
+                for (var i = 0; i < this.length; i++) {
+                    if (predicate.call(thisArg, this[i], i, this)) {
+                        return this[i];
+                    }
+                }
+            };
+        }
+        if (!Object.entries) {
+            Object.entries = function (obj) {
+                var keys = Object.keys(obj), iEnd = keys.length, entries = [];
+                for (var i = 0; i < iEnd; ++i) {
+                    entries.push([keys[i], obj[keys[i]]]);
+                }
+                return entries;
+            };
+        }
+        if (!Object.values) {
+            Object.values = function (obj) {
+                var keys = Object.keys(obj), iEnd = keys.length, values = [];
+                for (var i = 0; i < iEnd; ++i) {
+                    values.push(obj[keys[i]]);
+                }
+                return values;
+            };
+        }
+
+        (function () {
+            if (typeof window.CustomEvent === "function") return false;
+
+            function CustomEvent(event, params) {
+                params = params || { bubbles: false, cancelable: false, detail: undefined };
+                var evt = document.createEvent('CustomEvent');
+                evt.initCustomEvent(event, params.bubbles, params.cancelable, params.detail);
+                return evt;
+            }
+
+            CustomEvent.prototype = window.Event.prototype;
+
+            window.CustomEvent = CustomEvent;
+        })();
+
+    </script>
+
+    <script src="http://localhost:3031/es5/highcharts.src.js"></script>
```

Comment:
ES5-variant of `highcharts.src.js` includes the polyfills. If it is not working in IE11, the `ts/masters-es5/highcharts.src.ts` file has to be improved.